sherpa/bin/streaming_conformer_rnnt/streaming_server.py

```diff
@@ -180,18 +219,26 @@ def run_model_and_do_greedy_search(
     chunk_size = server.decode_chunk_size
     left_context = server.decode_left_context
     right_context = server.decode_right_context
+    decoding_method = server.decoding_method
 
     batch_size = len(stream_list)
 
     state_list = []
-    decoder_out_list = []
-    hyp_list = []
     feature_list = []
     processed_frames_list = []
+    if decoding_method == "greedy_search":
+        decoder_out_list = []
+        hyp_list = []
+    else:
+        rnnt_decoding_streams_list = []
+        rnnt_decoding_config = server.rnnt_decoding_config
     for s in stream_list:
+        if decoding_method == "greedy_search":
+            decoder_out_list.append(s.decoder_out)
+            hyp_list.append(s.hyp)
+        if decoding_method == "fast_beam_search":
```

Comment:
The name of this function is `run_model_and_do_greedy_search`.

Please move it to a separate function, e.g.,
`run_model_and_do_fast_beam_search`

Reply:
Oh, I did not notice this, will refactor it.

Reply:
I just changed the function name from `run_model_and_do_greedy_search` to `run_model_and_do_search`, I think it would be too much duplicate code if creating a separate function.